taxcalc/calcfunctions.py

```diff
@@ -590,86 +860,131 @@ def ItemDed(e17500_capped, e18400_capped, e18500_capped, e19200_capped,
-
-        ID_StateLocalTax_crt : Deduction for state and local taxes;
-                               ceiling as a decimal fraction of AGI
-
-        ID_RealEstate_crt : Deduction for real estate taxes;
-                            ceiling as a decimal fraction of AGI
-
-    Taxpayer Characteristics:
-        e17500_capped : Medical expenses, capped by ItemDedCap
-
-        e18400_capped : State and local taxes, capped by ItemDedCap
-
-        e18500_capped : Real-estate taxes, capped by ItemDedCap
-
-        e19200_capped : Interest paid, capped by ItemDedCap
-
-        e19800_capped : Charity cash contributions, capped by ItemDedCap
-
-        e20100_capped : Charity noncash contributions, capped by ItemDedCap
-
-        e20400_capped : Total miscellaneous expenses, capped by ItemDedCap
-
-        g20500_capped : Gross casualty or theft loss (before disregard),
-                        capped by ItemDedCap
+    Parameters
+    ----------
+    e17500_capped: float
+        Schedule A: medical expenses, capped as a decimal fraction of AGI
+    e18400_capped: float
+        Schedule A: state and local income taxes deductlbe, capped as a decimal fraction of AGI
+    e18500_capped: float
+        Schedule A: state and local real estate taxes deductible, capped as a decimal fraction of AGI
+    e19200_capped: float
+        Schedule A: interest deduction deductible, capped as decimal fraction of AGI
+    e19800_capped: float
+        Schedule A: charity cash contributions deductible, capped as a decimal fraction of AGI
+    e20100_capped: float
+        Schedule A: charity noncash contributions deductible, capped as a decimal fraction of AGI
+    e20400_capped: float
+        Schedule A: gross miscellaneous deductions deductible, capped as a decimal fraction of AGI
+    g20500_capped: float
+        Schedule A: gross causalty or theft loss deductible, capped as a decimal fraction of AGI
+    MARS: int
+        Filing marital status (1=single, 2=joint, 3=separate, 4=household-head, 5=widow(er))
+    age_head: int
+        Age in years of taxpayer
+    age_spouse: int
+        Age in years of spouse
+    c00100: float
+        Adjusted gross income (AGI)
+    c04470: float
+        Itemized deductions after phase out (0 for non itemizers)
+    c21040: float
+        Itemized deductions that are phased out
+    c21060: float
+        Itemized deductions before phase out (0 for non itemizers)
+    c17000: float
+        Schedule A: medical expenses deducted
+    c18300: float
+        Schedule A: state and local taxes plus real estate taxes deducted
+    c19200: float
+        Schedule A: interest deducted
+    c19700: float
+        Schedule A: charity contributions deducted
+    c20500: float
+        Schedule A: net causalty or theft loss deducted
+    c20800: float
+        Schedule A: net limited miscellaneous deductions deducted
+    ID_ps: list
+        Itemized deduction phaseout AGI start
```

Comment:
Add "(Pease)" to description to note relation to Pease Limitation -- also as in the `ID_prt` and `ID_crt` parameters as before.